api/src/main/resources/workbench.yaml

```diff
@@ -2842,9 +2858,46 @@ definitions:
       ACCESS_TO_CARE,
       EDUCATION_LEVEL,
       INCOME_LEVEL,
+      OTHER,
+      RACE_ASIAN,
+      RACE_AA,
+      RACE_HISPANIC,
+      RACE_AIAN,
+      RACE_MENA,
+      RACE_NHPI,
+      RACE_MORE_THAN_ONE,
+      AGE_CHILDREN,
+      AGE_ADOLESCENTS,
+      AGE_OLDER,
+      AGE_OLDER_MORE_THAN_75
+    ]
+
+  disseminateResearchEnum:
+    type: string
+    description: disseminate findings
```

Comment:
[nit] Either provide a meaningful description or leave this field blank. IMO this doesn't provide readers or maintainers any add'l info.